lib/install/data/data.go

```diff
@@ -70,6 +70,8 @@ type Data struct {
 
 	Force bool
 	UseRP bool
+
+	ScratchSize string
```

Comment:
Just out of curiousity, why is this a string while it's an `int` in `VirtualContainerHostConfigSpec`?